Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchyBasedLinkResolver.swift

```diff
@@ -67,9 +67,9 @@ final class PathHierarchyBasedLinkResolver {
     
     /// Map the resolved identifiers to resolved topic references for a given bundle's article, tutorial, and technology root pages.
     func addMappingForRoots(bundle: DocumentationBundle) {
-        resolvedReferenceMap[pathHierarchy.tutorialContainer.identifier] = bundle.tutorialsRootReference
```

Comment:
The root cause for this crash is that I had forgotten which is the "tutorial" page and which is the "technology" page so the tutorial link resolved the reference for the technology page and vice versa resulting in an assertion during rendering of the See Also section.

Reply:
should we rename these so that they are easier to distinguish?

Reply:
I think that's a good idea. There are still many places that use the "technology" terminology in the code and I think we should update them all to match the user-facing terminology.

I opened this issue about updating the general "technology" terminology in the code.